biogrid/apps/api/src/app/services/Biogrid.ts

```diff
@@ -40,21 +42,29 @@ function createRandomBuildingPosition(
 export async function simulateNewBiogrid(
   body: NewBiogridOpts
 ): Promise<BiogridSimulationResults> {
-  const buildings = new Array(body.numBuildings).map((v) => {
+  const buildings: Building[] = [];
+  for (let i = 0; i < body.numBuildings; i++) {
     const randomPos = createRandomBuildingPosition(
       body.townWidth,
       body.townHeight
     );
-    return new Building(10, randomPos.x, randomPos.y);
-  });
+    buildings.push(
+      new Building(
+        10,
```

Comment:
either add in-line comment on what the 10 is or use a constant like Roland suggested